serverless-functions/src/functions/common/helpers/retry-handler.private.js

```diff
@@ -39,6 +39,10 @@ exports.retryHandler = async (error, parameters, callback) => {
   const retryAttemptsMessage = attempts === 1 ? `${attempts} retry attempt` : `${attempts} retry attempts`;
   const message = errorMessage ? errorMessage : error;
 
+  // Perform retries only on retry-able response status codes
+  // 412: Received upon ETag conflict; retry without delay
+  // 429: Rate limited; retry with delay
+  // 503: Internal error; retry with delay
```

Comment:
This should be service unavailable not internal error

Reply:
Description was fetched from here https://www.twilio.com/docs/api/errors/503

Reply:
But is it not an http status code ?? 😂😂🤦‍♂️

Okay fair enough